src/spec/lib/jdk/JdkHelperSpec.ts

```diff
@@ -16,52 +16,47 @@
 
 'use strict';
 
-const JdkHelper = require('../../../lib/jdk/JdkHelper');
+import {JdkHelper} from '../../../lib/jdk/JdkHelper';
+import {Config} from '../../../lib/Config';
 
 describe('JdkHelper', () => {
   describe('getEnv()', () => {
     it('Creates the correct environment for Linux', () => {
-      const config = {
-        jdkPath: '/home/user/jdk8',
-      };
+      const config = new Config('/home/user/jdk8', '/home/user/sdktools');
       const process = {
         platform: 'linux',
         env: {
           'PATH': '',
         },
-      };
+      } as unknown as NodeJS.Process;
```

Comment:
What's this about?

Reply:
If we don't cast to `unknown` first, tsc will report the following error:
```
src/spec/lib/jdk/JdkHelperSpec.ts:40:23 - error TS2352: Conversion of type '{ platform: "darwin"; env: { 'PATH': string; }; }' to type 'Process' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
```